### ISSUE TITLE
Won't authenticate in DoneSSR

### ISSUE THREAD

**Issue by green3g** [canjs/can-connect-feathers#111]
[This line](https://github.com/canjs/can-connect-feathers/blob/master/session/session.js#L96) prevents can-connect from ever authenticating in done-ssr, therefore pages are rendered without authentication, even if user's cookies are provided to done-js.

**Comment by roemhildtg** [comment]
I'm not sure if we can just prevent setJWT from saving the internal state though.  `app.accessToken` gets set so that service hooks can run correctly on the client. If, for instance I prevent the app from ever setting `accessToken` none of my service requests will work because the hooks will prevent this. 

In my test app you can see the issue arise if you add this to the application.js set method:
(`public/node_modules/@feathersjs/feathers/lib/application.js:35`)
```javascript
    set (name, value) {
        if(name === 'accessToken' && window.doneSsr){
            debugger;
            return this;
        }
        this.settings[name] = value;
        return this;
    },
```

**Comment by matthewp** [comment]
If you add that does it fix the cookie issue?

**Comment by roemhildtg** [comment]
Yes, that does fix the issue, where users are getting mixed up. But donejs is unable to access any data from the api's since its not properly authenticated in feathers.

**Comment by roemhildtg** [comment]
Okay, I think I have a solution. Its a relatively simple fix in feathers-authentication.

In `populate-access-token`, 
```javascript
Object.assign(hook.params, { accessToken: app.get('storage').getItem(app.passport.options.storageKey) });
```

in `passport`:

remove references to app.set and app.get accessToken
